backend/job_descriptions/prompts.py

```diff
@@ -1,4 +1,53 @@
-JOB_DESCRIPTION_PROMPT = (
+# This module contains the prompt templates for generating job descriptions.
+# Main job description template
+JOB_DESCRIPTION_TEMPLATE = """
+Generate a detailed and professional job description for a **{job_title}** role at a U.S.-based tech company. Use a tone that is clear, structured, and appealing to experienced professionals.
+
+---
+
+**Guardrails:**
+- Do not invent company-specific details. Assume the company is a mid-to-large U.S. tech firm.
+- Reject or return a polite error if the input values (job title, custom note, or key skills) appear nonsensical or unrelated to a real job description.
+- Only generate content if the job title sounds like a legitimate role (e.g., “Software Engineer,” “Data Analyst”). If it's a placeholder or fake (e.g., “asdf”, “blah”), do not proceed.
+- Keep the tone professional, informative, and concise—avoid overly casual or buzzword-heavy language.
+- Follow the section structure exactly as provided.
+- Use bullet points for clarity, and avoid repetition.
+- Do not include salary information unless explicitly specified.
+- If the {custom_note} includes terms such as “undergrad student”, “opt student”, “cpt student”, or “f1 student” (not case-sensitive), include relevant notes.
```

Comment:
You probably don't want to include the "custom_note" input here. Rather, just saying if a custom note is included, do XYZ.


```suggestion
- If the "custom_note" includes terms such as “undergrad student”, “opt student”, “cpt student”, or “f1 student” (not case-sensitive), include relevant notes.
```